lib/discordrb/data/member.rb

```diff
@@ -247,10 +247,17 @@ def server_unmute
     end
 
     # Bans this member from the server.
-    # @param message_days [Integer] How many days worth of messages sent by the member should be deleted.
+    # @param message_days [Integer] How many days worth of messages sent by the member should be deleted. This will be deprecated in 4.0.
+    # @param message_seconds [Integer] How many seconds worth of messages sent by the member should be deleted.
     # @param reason [String] The reason this member is being banned.
-    def ban(message_days = 0, reason: nil)
-      server.ban(@user, message_days, reason: reason)
+    def ban(message_days = 0, message_seconds: nil, reason: nil)
```

Comment:
I don’t like `message_seconds` as a keyword argument, but I suppose it’s the only reasonable way to achieve being able to choose between units.